conf/modules.config

```diff
@@ -360,10 +360,21 @@ process {
         ]
     }
 
-    withName: 'NFCORE_RNAFUSION:BUILD_REFERENCES:STARFUSION_BUILD' {
+    withName: 'STARFUSION_BUILD' {
         cpus   = { 24    * task.attempt }
         memory = { 100.GB * task.attempt }
         time   = { 2.d  * task.attempt }
+        publishDir = [
+            enabled: !params.ctatsplicing && !params.all,
```

Comment:
Don't we want to enable publishing with params.ctatsplicing and params.all?

Reply:
No because `CTATSPLICING_PREPGENOMELIB` takes this reference directory and expands it, so there's no reason to publish them twice

Reply:
@rannick is this okay for you? :)